### PR TITLE
Update german language

### PR DESCRIPTION
**Pull request by Sound-Cat**

### CODE REVIEW

**Comment by dk-raw** [comment]
please use `yarn commit` or `yarn commit:signed`

File: kibbeh/public/locales/de/translation.json [hunk 1]
```diff
@@ -157,18 +157,18 @@
     },
     "micPermissionBanner": {
       "permissionDenied": "Wir haben keinen Zugriff auf dein Mikrofon. Öffne deine Browsereinstellungen, setze die nötigen Berechtigungen und lade die Seite erneut.",
-      "dismiss": "verstanden",
-      "tryAgain": "erneut versuchen"
+      "dismiss": "Verstanden",
+      "tryAgain": "Erneut Versuchen"
```
Comment (JKRhb) [inline kibbeh/public/locales/de/translation.json]:
```suggestion
      "tryAgain": "Erneut versuchen"
```

File: kibbeh/public/locales/de/translation.json [hunk 2]
```diff
@@ -157,18 +157,18 @@
...
         "public": "Öffentlich",
         "private": "Privat",
         "roomName": "Raumname",
         "roomDescription": "Raumbeschreibung",
-        "descriptionError": "die maximale Länge beträgt 500 Zeichen",
-        "nameError": "muss zwischen 2 und 60 Zeichen lang sein",
-        "subtitle": "Fill the following fields to start a new room"
+        "descriptionError": "Die maximale Länge beträgt 500 Zeichen",
+        "nameError": "Muss zwischen 2 und 60 Zeichen lang sein",
+        "subtitle": "Fülle die folgenden Felder aus um einen Raum zu erstellen"
```
Comment (JKRhb) [inline kibbeh/public/locales/de/translation.json]:
```suggestion
        "subtitle": "Fülle die folgenden Felder aus, um einen Raum zu erstellen"
```

File: kibbeh/public/locales/de/translation.json [hunk 3]
```diff
@@ -88,10 +88,10 @@
       "allowAllConfirm": "Bist du sicher? Damit erteilst du {{count}} Benutzern Redeberechtigungen."
     },
     "searchUser": { "search": "Suchen..." },
-    "soundEffectSettings": { "header": "Ton", "title": "Sound Settings" },
+    "soundEffectSettings": { "header": "Ton", "title": "Ton Einstellungen" },
```
Comment (JKRhb) [inline kibbeh/public/locales/de/translation.json]:
```suggestion
    "soundEffectSettings": { "header": "Ton", "title": "Toneinstellungen" },
```

File: kibbeh/public/locales/de/translation.json [hunk 4]
```diff
@@ -78,7 +78,7 @@
     "notFound": {
       "whoopsError": "Hoppla! Diese Seite ging im Gespräch verloren.",
       "goHomeMessage": "Keine Sorge. Du kannst",
-      "goHomeLinkText": "nach Hause"
+      "goHomeLinkText": "Nach Hause"
```
Comment (JKRhb) [inline kibbeh/public/locales/de/translation.json:81]: This should be lower case as it is appended to the `goHomeMessage` (I guess?)

```suggestion
      "goHomeLinkText": "nach Hause"
```

**Review by JKRhb — COMMENTED** [review]
Saw this PR by chance and wanted to suggest some small changes regarding capitalization and comma placement.

**Comment by amitojsingh366** [comment]
please follow the proper pattern for commit messages *(💡TIP: you can use `yarn commit` or `yarn commit:signed` and answer the questions asked to generate a properly formatted one for you!)*

to fix your most recent commit you can run the command `git commit --amend -m "properly formatted commit message here"`

to fix your older/multiple commits please take a look at [this article](https://docs.github.com/en/github/committing-changes-to-your-project/changing-a-commit-message#amending-older-or-multiple-commit-messages)

**Comment by Sound-Cat** [comment]
New Pull Request soon

**Pull request closed without merging**